dev/manual_tests/lib/menu_anchor.dart

```diff
@@ -211,10 +211,7 @@ class _ControlsState extends State<_Controls> {
             alignmentOffset: const Offset(100, -8),
             menuChildren: <Widget>[
               MenuItemButton(
-                shortcut: const SingleActivator(
-                  LogicalKeyboardKey.keyB,
-                  control: true,
-                ),
+                shortcut: TestMenu.standaloneMenu1.shortcut,
```

Comment:
I use the `manual_tests` folder to put any tests that I use when validating an implementation.  Basically, like an example, but instead of something "reasonable", I try and make it use all of the features of a widget or system.  Then I can try them out while I am developing, or use them to reproduce problems when they are reported. So, not just keyboard and shortcut events, but all kinds of things.

Reply:
Got it! Never used this folder before, thanks for the explanation:)